lib/di/app_injector.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:data/di/data_injector.dart';
```

Comment:
due to the fact that you made a modular architecture in the master branch, I cannot leave comments on some points of this "PR", namely:
- pubspec.yaml: remove domain;
- pubspec.yaml(data): remove get_it;

Reply:
removed